### ISSUE TITLE
How to remove inline "style" attribute

### ISSUE THREAD

**Issue by antonybritto** [react-icons/react-icons#124]
When i try to import fontawesome icons i am getting "style='vertical-align:middle'" attribute on the SVG tags. Since i am creating AMP style tags are not allowed, hence wanna remove it. Do i need to create a HOC? Any help here will be appreciated.

**Comment by luisherranz** [comment]
We are having the same problem here when using React to render amp html.

**Comment by antonybritto** [comment]
@luisherranz - For hotfix, currently i am doing a nasty fix which does string replace("style='vertical-align:middle'", "") in the final react rendered HTML.

**Comment by luisherranz** [comment]
We've done a PR to fix this problem:
https://github.com/gorangajic/react-icon-base/pull/30

It adds a new prop called `verticalAlign` which can be used to remove the inline style:

```js
<Icon verticalAlign='none' />
```

In the meantime, anyone wanting to remove the inline style attribute can use our fork with:

```
npm install git+https://github.com/wp-pwa/react-icons.git#fix-vertical-align-with-lib
```

**Comment by hellochar** [comment]
see https://github.com/react-icons/react-icons/issues/68

**Comment by luisherranz** [comment]
What do you mean?

**Comment by luisherranz** [comment]
Doesn't work for me:
https://codesandbox.io/s/jzymx64k1w

Both `<FaBeer />` and `<FaBeer style={{}} />` produce the same HTML output:
```html
<svg fill="currentColor" preserveAspectRatio="xMidYMid meet" height="1em" width="1em" viewBox="0 0 40 40" style="vertical-align: middle;">
```

Am I still doing something wrong?

**Comment by luisherranz** [comment]
I have added a couple of tests to the codesandbox:
[![Edit React icons verticalAlign problem (used as example)](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/jzymx64k1w?module=%2Fsrc%2Findex.spec.js)

**Comment by kamijin-fanta** [comment]
fix in react-icons@3.0.0 / pr: #141
It can be set with the context API of React 16.3. try new version!
see: https://github.com/react-icons/react-icons#configuration
